### PR TITLE
Update node/pnpm versions in README.md

### PR DESCRIPTION
**Pull request by pjeweb**

### CODE REVIEW

**Comment by MattIPv4** [comment]
Is it worth just updating the text in the readme to say check the package.json engines? So we don't have the info duplicated (so we don't forget to update it again lol)

File: README.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ This project uses pnpm workspaces. The main app is the website package (`apps/we
 
 For development:
 
-- node v16
-- pnpm 7 with workspaces
+- node v18
+- pnpm 8 with workspaces
```
Comment (MattIPv4) [inline README.md:18]:
```suggestion
- Node.js
- pnpm with workspaces
```

File: README.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ TODO
 
 ### Local development
 
-1. Install Node.js v16 and pnpm
+1. Install Node.js and pnpm
```
Comment (MattIPv4) [inline README.md:64]:
```suggestion
1. Install Node.js and pnpm (see `engines` in `package.json`)
```

**Comment by pjeweb** [comment]
> Is it worth just updating the text in the readme to say check the package.json engines? So we don't have the info duplicated (so we don't forget to update it again lol)

yeah that makes sense. i also tried to spell all the names properly

**Review by MattIPv4 — APPROVED** [review]

**Pull request merged**